lib/slack-ruby-bot/rspec/support/slack-ruby-bot/it_behaves_like_a_slack_bot.rb

```diff
@@ -1,15 +1,49 @@
 # frozen_string_literal: true
 
 shared_examples 'a slack ruby bot' do
```

Comment:
Curious, is the shared example so it can be used by subclasses of Server? What was the thinking behind separating App and Server?

Reply:
Exactly. An app is something that builds on top of server and does custom things on hooks.